### ISSUE TITLE
Can't install

### ISSUE THREAD

**Issue by LeVi1242** [ORelio/Minecraft-Console-Client#631]
I am not able to install the client i get the error 
```
Unhandled Exception:
System.InvalidOperationException: object_op ---> System.BadImageFormatException: Method has no body
  at System.IO.CoreFX.FileSystemWatcher.StartRaisingEvents () [0x00024] in <a9a08e39ba304bd0a84c49bd158dfc02>:0 
  at System.IO.CoreFX.FileSystemWatcher.StartRaisingEventsIfNotDisposed () [0x0001c] in <a9a08e39ba304bd0a84c49bd158dfc02>:0 
  at System.IO.CoreFX.FileSystemWatcher.set_EnableRaisingEvents (System.Boolean value) [0x0002a] in <a9a08e39ba304bd0a84c49bd158dfc02>:0 
  at (wrapper remoting-invoke-with-check) System.IO.CoreFX.FileSystemWatcher.set_EnableRaisingEvents(bool)
  at System.IO.CoreFXFileSystemWatcherProxy.<StartDispatching>m__0 (System.IO.CoreFX.FileSystemWatcher internal_fsw, System.IO.FileSystemWatcher fsw) [0x00048] in <a9a08e39ba304bd0a84c49bd158dfc02>:0 
  at System.IO.CoreFXFileSystemWatcherProxy.Operation (System.Action`4[T1,T2,T3,T4] map_op, System.Action`2[T1,T2] object_op, System.Object handle, System.Action`2[T1,T2] cancel_op) [0x001d3] in <a9a08e39ba304bd0a84c49bd158dfc02>:0 
   --- End of inner exception stack trace ---
  at System.IO.CoreFXFileSystemWatcherProxy.Operation (System.Action`4[T1,T2,T3,T4] map_op, System.Action`2[T1,T2] object_op, System.Object handle, System.Action`2[T1,T2] cancel_op) [0x001f8] in <a9a08e39ba304bd0a84c49bd158dfc02>:0 
  at System.IO.CoreFXFileSystemWatcherProxy.StartDispatching (System.Object handle) [0x00007] in <a9a08e39ba304bd0a84c49bd158dfc02>:0 
  at System.IO.FileSystemWatcher.Start () [0x00024] in <a9a08e39ba304bd0a84c49bd158dfc02>:0 
  at System.IO.FileSystemWatcher.set_EnableRaisingEvents (System.Boolean value) [0x00049] in <a9a08e39ba304bd0a84c49bd158dfc02>:0 
  at (wrapper remoting-invoke-with-check) System.IO.FileSystemWatcher.set_EnableRaisingEvents(bool)
  at MinecraftClient.Protocol.Session.SessionCache.InitializeDiskCache () [0x00050] in <d9cbc5e1dac242a589da951d2d7f2f72>:0 
  at MinecraftClient.Program.Main (System.String[] args) [0x0020e] in <d9cbc5e1dac242a589da951d2d7f2f72>:0 
[ERROR] FATAL UNHANDLED EXCEPTION: System.InvalidOperationException: object_op ---> System.BadImageFormatException: Method has no body
  at System.IO.CoreFX.FileSystemWatcher.StartRaisingEvents () [0x00024] in <a9a08e39ba304bd0a84c49bd158dfc02>:0 
  at System.IO.CoreFX.FileSystemWatcher.StartRaisingEventsIfNotDisposed () [0x0001c] in <a9a08e39ba304bd0a84c49bd158dfc02>:0 
  at System.IO.CoreFX.FileSystemWatcher.set_EnableRaisingEvents (System.Boolean value) [0x0002a] in <a9a08e39ba304bd0a84c49bd158dfc02>:0 
  at (wrapper remoting-invoke-with-check) System.IO.CoreFX.FileSystemWatcher.set_EnableRaisingEvents(bool)
  at System.IO.CoreFXFileSystemWatcherProxy.<StartDispatching>m__0 (System.IO.CoreFX.FileSystemWatcher internal_fsw, System.IO.FileSystemWatcher fsw) [0x00048] in <a9a08e39ba304bd0a84c49bd158dfc02>:0 
  at System.IO.CoreFXFileSystemWatcherProxy.Operation (System.Action`4[T1,T2,T3,T4] map_op, System.Action`2[T1,T2] object_op, System.Object handle, System.Action`2[T1,T2] cancel_op) [0x001d3] in <a9a08e39ba304bd0a84c49bd158dfc02>:0 
   --- End of inner exception stack trace ---
  at System.IO.CoreFXFileSystemWatcherProxy.Operation (System.Action`4[T1,T2,T3,T4] map_op, System.Action`2[T1,T2] object_op, System.Object handle, System.Action`2[T1,T2] cancel_op) [0x001f8] in <a9a08e39ba304bd0a84c49bd158dfc02>:0 
  at System.IO.CoreFXFileSystemWatcherProxy.StartDispatching (System.Object handle) [0x00007] in <a9a08e39ba304bd0a84c49bd158dfc02>:0 
  at System.IO.FileSystemWatcher.Start () [0x00024] in <a9a08e39ba304bd0a84c49bd158dfc02>:0 
  at System.IO.FileSystemWatcher.set_EnableRaisingEvents (System.Boolean value) [0x00049] in <a9a08e39ba304bd0a84c49bd158dfc02>:0 
  at (wrapper remoting-invoke-with-check) System.IO.FileSystemWatcher.set_EnableRaisingEvents(bool)
  at MinecraftClient.Protocol.Session.SessionCache.InitializeDiskCache () [0x00050] in <d9cbc5e1dac242a589da951d2d7f2f72>:0 
  at MinecraftClient.Program.Main (System.String[] args) [0x0020e] in <d9cbc5e1dac242a589da951d2d7f2f72>:0 
```

I think part  of the reason this is happening is because i am using kali linux please can you fix this asap?

**Comment by ORelio** [comment]
Hi,

This error happens in your Mono/.NET framework, not MCC. Try reinstalling/upgrading it.

Alternatively, to avoid calling the faultly function in your framework, you can try setting sessioncache=memory in your INI file.

**Comment by ORelio** [comment]
Fixed as per #684
